aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -119,15 +105,26 @@ private void MakeScene()
             if (_amountOfRooms > _gridWidthX * _gridWidthZ)
                 throw new Exception("The amount of rooms is larger than the available places in the grid.");
 
-            if (_useSeed) _random = new Random(_seed);
+            if (_useSeed)
+            {
+                _random = new Random(_seed);
+            }
+            else
+            {
+                Random tempRandom = new();
+                _seed = tempRandom.Next();
+                _random = new Random(_seed);
+            }
```

Comment:
Use conditional inverting to simplify this code.
```suggestion
            if (!_useSeed)
            {
                Random tempRandom = new();
                _seed = tempRandom.Next();
            }
            _random = new Random(_seed);
            
```